src/gdphelper/gdpplotter.py

```diff
@@ -12,44 +12,40 @@ def gdpplotter(cleaned_Dataframe, aggregation):
     ----------
     cleaned_Dataframe : pandas.DataFrame
         This is the name of the dataset that you want to draw a chart. It is an output from the function gdpcleaner
-    aggregation : string
-        There are two types of aggregation, "canada" and "province".
-        The first one will give the GDP by total Canada.
-        The first one will give the GDP by province.
-        
+    
     Returns
     -------
-        line chart of GDP by the Region VS periods.
+        chart of GDP by the Region according to the information within the DataFrame
 
     Examples
+    >>> url = "https://www150.statcan.gc.ca/n1/tbl/csv/36100408-eng.zip"
+    >>> df, x = gdpimporter(url)
+    >>> cleaned_df = gdpcleaner(df)
+    >>> gdpplotter(cleaned_df)
     
-    >>>df = pd.read_csv("36100408.csv")
-    >>>aggregation = "canada"
-    >>>cleaned_df = gdpcleaner(df)
-    >>>gdpplotter(cleaned_df, aggregation)
 
     """
     # check for DataFrame input argument
     if (type(cleaned_Dataframe) != pd.DataFrame):
          raise TypeError("Argument must be a specific Pandas DataFrame, use the 'gpdcleaner' to create this dataFrame")
     
-    # checking the inputs of the function: aggregation
-    if aggregation in ["canada", "province"]:
-        pass
-    else:
-        raise ValueError(
-            'The aggragation should be "canada" or "province".'
-        )
-
     # reading the database
     df = cleaned_Dataframe
-    
-    # transforming the database to fit on matplolib's data structure.
-    
-    # using the same unit for the charts: millions
-    unit= ['millions', 'units ']
-    df_filtered = df[df['Scale'].isin(unit)]
-    
+
+    #cheking units in Scale
+    units_type = df['Scale'].unique()
+
+    # using the same unit for the charts
+    if 'millions' in units_type:
+        df_filtered = df[df['Scale'] == 'millions']
+    elif 'units' in units_type:
+        df_filtered = df[df['Scale'] == 'units']
+    elif 'units ' in units_type:
+        df_filtered = df[df['Scale'] == 'units ']
+    else:
+        raise TypeError("This dataframe don't have the GDP's units 'millions' or 'units ' or 'units' which are allowed for this analysis. Please check your DataFrame")
```

Comment:
@aldojasb this will fix the problem with the %. I suggest we all change the example in the docstring to match a data file that doesn't have % in units